readme.md

```diff
@@ -42,40 +48,40 @@ console.log(await getStream(stream));
 
 ## API
 
-The methods returns a promise that resolves when the `end` event fires on the stream, indicating that there is no more data to be read. The stream is switched to flowing mode.
+The following methods read the stream's contents and return it as a promise.
 
 ### getStream(stream, options?)
 
-Get the given `stream` as a string.
-
-#### options
-
-Type: `object`
+`stream`: [`stream.Readable`](https://nodejs.org/api/stream.html#class-streamreadable)
+`options`: [`Options`](#options)
 
-##### maxBuffer
-
-Type: `number`\
-Default: `Infinity`
-
-Maximum length of the returned string. If it exceeds this value before the stream ends, the promise will be rejected with a `MaxBufferError` error.
+Get the given `stream` as a string.
 
 ### getStreamAsBuffer(stream, options?)
 
 Get the given `stream` as a buffer.
 
-It honors the `maxBuffer` option as above, but it refers to byte length rather than string length.
-
 ```js
 import {getStreamAsBuffer} from 'get-stream';
 
 const stream = fs.createReadStream('unicorn.png');
-
 console.log(await getStreamAsBuffer(stream));
 ```
 
+#### options
```

Comment:
Move options below since a new method will be added for web streams, which will have the same signature as the other ones.